content/news/2020_06_28_github_actions.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "New GRASS GIS source code analysis with GitHub Actions"
+date: 2020-06-28T12:12:12+01:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### New GRASS GIS source code analysis with GitHub Actions
+
+After the successful migration from SVN to git earlier this year ([here](/news/2019_05_17_svn_to_github_migration_started/) and [here](/news/2020_01_23_grass_gis_issues_now_on_github/)), a new feature has been implemented and enabled: [GRASS GIS source code analysis with GitHub Actions](https://github.com/OSGeo/grass/actions) to build (compile), test, and perform Python static code analysis, in order to improve the overall quality, maintainability and standard compliance of our code.
+
+The C and C++ source code is checked with GCC C/C++ standard checks and additionally with gnu99 and c++11 tests. Python code, on the other hand, is tested with a static code analysis/code quality check/linting using Flake8. The workflow builds and tests GRASS GIS on Ubuntu 16.04, 18.04 and Centos 7. An additional OSGeo4W workflow for GitHub Actions is currently in preparation. Success, errors and warnings are reported accordingly in the GitHub Actions interface for each pull request.
```

Comment:
```suggestion
The C and C++ source code is checked with GCC C/C++ standard checks and additionally with gnu99 and c++11 tests. Python code, on the other hand, is tested with a static code analysis/code quality check/linting using Flake8. The workflow builds and tests GRASS GIS on Ubuntu 16.04, 18.04, Centos 7 as well as OSGeo4W. Success, errors and warnings are reported accordingly in the GitHub Actions interface for each pull request.
```

Reply:
... like this?

Reply:
Looks good!

Reply:
> The C and C++ source code is checked with GCC C/C++ standard checks and additionally with gnu99 and c++11 tests. 

Notably, we don't check yet ISO or strict compliance, it is so far just a rough check, so saying something like *GCC C/C++ checks including GNU C11 and C++11* is probably better.

> Python code, on the other hand, is tested with a static code analysis/code quality check/linting using Flake8.

...with tens of messages disabled. Can you please add a note about seeking contributors to reduce the number of remaining issues?

Reply:
Done so, thanks for the suggestions